src/SprykerEco/Zed/CrefoPay/Persistence/Propel/Schema/spy_payment_crefo_pay.schema.xml

```diff
@@ -53,7 +53,7 @@
             <reference foreign="id_sales_order_item" local="fk_sales_order_item"/>
         </foreign-key>
 
-        <unique name="spy_payment_crefo_pay_order_item-unique-payment_crefo_pay-order_item">
+        <unique name="spy_payment_crefo_pay_order_item-unique-payment-order_item">
```

Comment:
BC break 

see https://spryker.atlassian.net/wiki/spaces/CORE/pages/3048767586/Is+my+module+change+major

Reply:
Fixed.